CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.7.3 – 2025-04-03
+
+1. Add more information to `$feature_flag_called` events for `/decide` requests such as flag id, version, reason, and the request id.
```

Comment:
this is was a minor version bump for other projects, probably worth the same here.